### ISSUE TITLE
Not all Staking Modes are shown

### ISSUE THREAD

**Issue by cyril2000** [aaronk6/Kraken-MoneyMoney#17]
Staked assets that have more than one Staking mode (like Solana) are not shown in the list.

List of Asset-Names as shown in exported CSV-files:
```
ATOM21.S

SOL.S
SOL03.S

KAVA.S
KAVA21.S

DOT.S
DOT28.S
```

**Comment by dvcrn** [comment]
Yeah it looks ike the script is only fetching the base staking/earn, not all pairs

**Comment by aaronk6** [comment]
Thanks for the report! My time is limited at the moment and I don’t own any staked assets, so I can’t promise when I get around fixing this. Maybe @zacczakk is able to help out here?
